packages/integration-tests/Granules.js

```diff
@@ -75,6 +77,17 @@ const getGranuleWithStatus = async (params = {}) =>
     }
   );
 
+const buildRandomizedGranule = (overrides = {}) => ({
```

Comment:
Yeah, we don't, I wasn't sure it was ok to pull it from there because I saw buildRandomizedCollections in the @cumulus/integration-test package https://github.com/nasa/cumulus/blob/master/packages/integration-tests/Collections.js#L80

Reply:
Yeah, we're not exactly being consistent. We can leave this

Reply:
I'm fine yanking it if you think it's better to pull it from inside the api/lib/testUtils.  I don't love duplicate code. But also just wasn't sure what the team wants to do with those.

Reply:
Yeah we could pull from api/lib/testUtils but I don't love that either. I don't love treating our `api` package as the source of any helpers. So the duplication here doesn't bother me too much

Reply:
Well, Jenny followed me down this road and added some for executions.  I'm not a fan and I'm going to pull from testUtils in my next follow-on PR I think to get rid of this.